.drone/lcov-report.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+set -e
+# set -x
+
+SKIPLIST="math"
+
+: "${LCOV_SKIP_PATTERN:='^[9]'}" # Set default lcov skip pattern
+
+pwd
+
+export CODECOV_SCRIPT=${BOOST_CI_SRC_FOLDER}/ci/travis/codecov.sh
+export CI_DIR=${BOOST_CI_SRC_FOLDER}/ci
+export BOOST_CI_CODECOV_IO_UPLOAD="skip"
```

Comment:
Why set this and call `codecov.sh "upload"`? I guess you just want `codecov.sh "collect"`

Reply:
Switched to "collect".